sparse/src/KokkosSparse_CrsMatrix.hpp

```diff
@@ -342,7 +342,7 @@ class CrsMatrix {
   typedef SizeType size_type;
 
   //! Type of a host-memory mirror of the sparse matrix.
-  typedef CrsMatrix<ScalarType, OrdinalType, host_mirror_space, MemoryTraits, SizeType> HostMirror;
+  typedef CrsMatrix<ScalarType, OrdinalType, host_mirror_space, MemoryTraits, SizeType> host_mirror_type;
```

Comment:
The Kokkos PR kept ``HostMirror`` as a deprecated type, maybe our matrix types should have that too for consistency.

Reply:
Good point, I'll update accordingly, thanks @brian-kelley